app/models/manageiq/providers/base_manager/refresher.rb

```diff
@@ -43,7 +43,7 @@ def refresh
           rescue => e
             raise if EmsRefresh.debug_failures
 
-            _log.error("#{log_ems_target} Refresh failed")
+            _log.error("#{log_ems_target} Refresh failed: #{e}")
```

Comment:
I wonder if we need to truncate the message.  I remember https://github.com/ManageIQ/manageiq/pull/16550 and this bz: https://bugzilla.redhat.com/show_bug.cgi?id=1506718, where the firewall blocked the request and threw back a html response in the exception message.

Reply:
I wonder if we could do this more generally in the Logger?  We log exceptions everywhere...

Reply:
I believe we limit them to [1 megabyte](https://github.com/ManageIQ/manageiq-gems-pending/blob/ea1f981d4c4b041264e5849185d07e9766816164/lib/gems/pending/util/vmdb-logger.rb#L163).  We typically truncate it smaller if we know the area we're logging should have a small message.

Reply:
Okay done